src/Core/tests/DeviceTests/Handlers/Label/LabelHandlerTests.iOS.cs

```diff
@@ -23,5 +25,34 @@ Task ValidateNativeBackgroundColor(ILabel label, Color color)
 				GetNativeLabel(CreateHandler(label)).AssertContainsColor(color);
 			});
 		}
+
+		UITextAlignment GetNativeTextAlignment(LabelHandler labelHandler) =>
+			GetNativeLabel(labelHandler).TextAlignment;
+
+		[Fact(DisplayName = "[LabelHandler] HorizontalTextAlignment Updates Correctly")]
+		public async Task HorizontalTextAlignmentInitializesCorrectly()
```

Comment:
Are you planning on adding a `VerticalTextAlignmentInitializesCorrectly()` iOS test too?